pkg/compiler/codegen.go

```diff
@@ -733,6 +713,7 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 			emitOpcode(c.prog.BinWriter, opcode.DROP)
 			emitOpcode(c.prog.BinWriter, opcode.THROW)
 		} else if isStringType(c.typeInfo.Types[arg].Type) {
+			ast.Walk(c, arg)
```

Comment:
Does this line really belong to 1dee3264c98378fe01beff72517909b0670e55cb?

Reply:
umm, it is already in 1dee326 ?